src/styles/utils/veriables.scss

```diff
@@ -0,0 +1,7 @@
+$card-width: 200px;
```

Comment:
The file name 'veriables.scss' contains a typo. It should be 'variables.scss'. Correct the file name to ensure that all import statements in other SCSS files work correctly.